app/src/androidTest/java/com/example/triptracker/userProfile/UserProfileListTest.kt

```diff
@@ -0,0 +1,114 @@
+package com.example.triptracker.userProfile
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.example.triptracker.model.profile.UserProfile
+import com.example.triptracker.model.profile.UserProfileList
+import java.util.Date
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class UserProfileListTest {
+  private val userProfileList1 = UserProfileList(listOf())
+  private val userProfileList2 =
+      UserProfileList(
+          listOf(
+              UserProfile(
+                  id = "1",
+                  name = "Alice",
+                  surname = "Smith",
+                  birthdate = Date(),
+                  pseudo = "AliceS"),
+              UserProfile(
+                  id = "2", name = "Bob", surname = "Johnson", birthdate = Date(), pseudo = "BobJ"),
+              UserProfile(
+                  id = "3", name = "Bob", surname = "Brown", birthdate = Date(), pseudo = "BobB")))
+
+  private val userProfileList3 =
+      UserProfileList(
+          listOf(
+              UserProfile(
+                  id = "4",
+                  name = "Alice",
+                  surname = "Smith",
+                  birthdate = Date(),
+                  pseudo = "AliceS"),
+              UserProfile(
+                  id = "5", name = "Bob", surname = "Johnson", birthdate = Date(), pseudo = "BobJ"),
+              UserProfile(
+                  id = "6",
+                  name = "Charlie",
+                  surname = "Brown",
+                  birthdate = Date(),
+                  pseudo = "CharlieB")))
+
+  //    @Test
+  //    fun getAllUserProfilesTest() {
```

Comment:
Is this a test you want to use later? If not, it might be better to just remove it

Reply:
I added a TODO comment to make it clearer that it will be used later

Reply:
Cool! Thanks :)